pkgs/tools/backup/tsm-client/default.nix

```
@@ -1,15 +1,20 @@
 { lib
+, callPackage
+, nixosTests
 , stdenv
+, fetchurl
 , autoPatchelfHook
+, rpmextract
+, openssl
+, zlib
+, lvm2  # LVM image backup and restore functions (optional)
+, acl ? null  # EXT2/EXT3/XFS ACL support (optional)
```

```suggestion
, acl # EXT2/EXT3/XFS ACL support (optional)
```
``? null`` is not required for optional features.

---

Oops, I missed that one.